app/assets/javascripts/discourse/app/controllers/static.js

```diff
@@ -20,6 +21,9 @@ export default Controller.extend({
     );
   },
 
+  @discourseComputed
+  wavingHandURL: () => wavingHandURL(),
+
```

Comment:
@ellaestigoy Where is this used?

Reply:
The 👋🏼 is used when they don't have a logo uploaded